src/test/scala/io/iohk/ethereum/jsonrpc/JsonRpcControllerFixture.scala

```diff
@@ -113,6 +115,11 @@ class JsonRpcControllerFixture(implicit system: ActorSystem)
     unixTimestamp = 0
   )
 
+  val checkpoint = ObjectGenerators.fakeCheckpointGen(2, 5).sample.get
+  val blockWithCheckpoint = CheckpointingTestHelpers.createBlockWithCheckpoint(Fixtures.Blocks.Block3125369.header, checkpoint)
```

Comment:
I thought we had removed this as it's duplicated with `CheckpointBlockGenerator`

Reply:
I think it's my bad that it wasn't removed in the end. But most of its usages have been replaced with CheckpointBlockGenerator so it should be easy to remove now